x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/inventory_metric_threshold_executor.ts

```diff
@@ -124,6 +124,7 @@ export const createInventoryMetricThresholdExecutor = (libs: InfraBackendLibs) =
         })
       )
     );
+
```

Comment:
You could revert this change to avoid the code owner review from Infra UI team 👍🏼